tests/box_data_test.cpp

```diff
@@ -15,9 +15,9 @@ box_data_test_float()
 {
   ASSERT_VIRTUAL_MEMORY(*virtual_memory, 0);
 
-  float fnum = 65.0f;
+  float64_t fnum = 65.0f;
```

Comment:
change to 65.0

Reply:
change 65.0f to 65.0 because literal 65.0f spawns float literal which is then turned to double. That is avoided by omitting "f".